Formula/texlab.rb

```diff
@@ -0,0 +1,34 @@
+class Texlab < Formula
+  desc "Implementation of the Language Server Protocol for LaTeX"
+  homepage "https://texlab.netlify.com/"
+  url "https://github.com/latex-lsp/texlab/archive/v1.8.0.tar.gz"
+  sha256 "af644d2555c3852513135e87dc6f9bc8b5ee789a4f1c151f4478d108fa007c49"
+  head "https://github.com/latex-lsp/texlab.git"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--locked",
```

Comment:
This can be a single line, no need to split it I think.

Reply:
I followed other formulas, but I can change it of course.